polyply/src/gen_itp.py

```diff
@@ -80,9 +82,11 @@ def gen_params(args):
 
     # Raise warning if molecule is disconnected
     if not nx.is_connected(meta_molecule.molecule):
-        n_components = len(list(nx.connected_components(meta_molecule.molecule)))
-        msg = "You molecule consists of {:d} disjoint parts. Perhaps links were not applied correctly."
-        LOGGER.warning(msg, (n_components))
+        LOGGER.warning("Your molecule consists of disjoint parts."
+                       "Perhaps links were not applied correctly.")
+        msg = "Missing link between residue {idxA} {resA} and residue {idxB} {resB}"
+        for missing in find_missing_edges(meta_molecule, meta_molecule.molecule):
+            LOGGER.warning(msg, **missing)
```

Comment:
I would combine these to a single warning

Reply:
This is on purpose because the GUI picks up the individual lines from the sdt-err/std-out and then uses the information given to highlight missing links.